aplib.net-demo/Assets/Scripts/MainMenu/CanvasManager.cs

```diff
@@ -144,60 +126,82 @@ public void OnPlayerDeath(HealthComponent _)
     /// <summary>
     /// When the play button in the main menu is clicked, it wil teleport the player to the game.
     /// </summary>
-    public void PlayGame() => SceneManager.LoadScene(_sceneNameGame);
+    public void PlayGame()
+    {
+        SceneManager.LoadScene(_sceneNameGame);
+    }
 
     /// <summary>
     /// When the Quit To Menu button in the game setting is clicked, it will take you back to the menu.
     /// </summary>
-    public void ToMenu() => SceneManager.LoadScene(_sceneNameStartingMenu);
+    public void ToMenu()
+    {
+        SceneManager.LoadScene(_sceneNameStartingMenu);
+    }
 
     /// <summary>
     /// This button toggles the menu canvas off and the setting canvas on or game settings on and off.
     /// </summary>
     public void OnToggleSettings()
     {
-        if (_preventSettingsToggle) return;
+        _isOnSettings = !_isOnSettings;
 
-        if (_currentSceneName == _sceneNameStartingMenu) // Toggle from menu to menu settings and back.
+        if (_currentSceneName == _sceneNameStartingMenu)
```

Comment:
You could rewrite this if else if else if statement using a `switch` statement. On other places as well

Reply:
Done this as well!